### ISSUE TITLE
Custom theming [light/dark mode]

### ISSUE THREAD

**Issue by mosesokemwa** [tunapanda/wikonnect#94]
As a user, I can select to view wikonnect as either dark mode or light mode
- [ ] styling for the dark mode
- [ ] styling for light mode

**Comment by macharian8** [comment]
I'm working on this

**Comment by KingKimani** [comment]
I would love to do work on this

**Comment by 0xMurage** [comment]
@KingKimani kindly go ahead. If you need any assistance, kindly reach out to the team on the Wikonnect discord channel https://discord.gg/jeqS2p7X. We appreciate your time and dedicated effort.

**Comment by i1d9** [comment]
@KingKimani I want to help

**Comment by Wambua13** [comment]
I want to contribute on this.

**Comment by proverbial-ninja** [comment]
@0yah @KingKimani , I suggest the first step should be some sort of documentation on the implementation plan ? 
maybe check into our discord and we could have a chat about it ... I've created a  channel for this issue
https://discord.gg/ktQeK8Kj

**Comment by KingKimani** [comment]
no problem, do u have any plan in mind @proverbial-ninja ?

**Comment by Wambua13** [comment]
Yeah we should have documentation first. I am a newbie so I need lots of
research but I know I will be productive.
